cloudpickle/cloudpickle.py

```diff
@@ -354,23 +367,14 @@ def inner(value):
                 co.co_name,
                 co.co_firstlineno,
                 co.co_lnotab,
-                co.co_cellvars,  # this is the trickery
-                (),
+                co.co_cellvars,  # co_freevars is initialized with co_cellvars
+                (),  # co_cellvars is made empty
             )
-
-_cell_set_template_code = _make_cell_set_template_code()
```

Comment:
Why not keep this? Creating a new `_cell_set_template_code` instance each time we call `_cell_set` might lead to some performance regression, no? Could you try to run a micro-benchmark to check whether this performance impact is noticeable or not?

Reply:
Yes indeed. On pickling this function
```python
def f():
    a = 1
    def g():
        return a
    return g

g = f()
```
this PR is 2.5% slower than master.

Reply:
Thanks.